frontend/components/campaigns/dashboard/CampaignDateInfoIcon.tsx

```diff
@@ -0,0 +1,20 @@
+import CalendarTodayIcon from '@mui/icons-material/CalendarToday';
+import { Typography } from '@mui/material';
+import { Stack } from '@mui/system';
+import { Moment } from 'moment';
+import { campaignDateIconSx } from '../../../styles/components/campaigns/dashboard/CampaignDashboardStyles';
+import { DATE_FORMAT } from '../../../utils/constants';
+
+interface CampaignDateInfoProps {
+  date: Moment;
+}
+
+const CampaignDateInfoIcon = ({ date }: CampaignDateInfoProps) => (
+  <Stack sx={campaignDateIconSx}>
+    <CalendarTodayIcon fontSize="large" />
+
+    <Typography>{date.format(DATE_FORMAT)}</Typography>
```

Comment:
Can we have a user facing date format that uses MMM (Like Jan, Feb, Mar). The reason why I used MM previously is because the MUI date input fields (for forms) doesn't accept MMM.